modules/HardwareDrivers/PowerMeters/LemDCBM400600/tests/test_lem_dcbm_400600_controller.cpp

```diff
@@ -244,8 +244,8 @@ TEST_P(LemDCBM400600ControllerTestInvalidResponses, test_start_transaction_fail_
     // Verify
     EXPECT_EQ(transaction_request_status_to_string(res.status), "UNEXPECTED_ERROR");
     EXPECT_THAT(res.error.value(), testing::MatchesRegex("Failed to start transaction mock_transaction_id.*"));
```

Comment:
Before that I would add a line like that to check if `res.error` has really a value:
`EXPECT_TRUE(res.error.has_value());`